features.yaml

```diff
@@ -0,0 +1,55 @@
+features:
+  - name: "Protocols:HTTP1.1/HTTP2/gRPC/TCP"
+    level:
+      - checklist:
+        maturity: Stable
+        nextExpectedPromotion:
+    deprecation:
```

Comment:
Can we leave deprecation block off if the feature is active? OK to do in follow up since I want to get this in.

Reply:
agree with Sven's comment on deprecation.  I think `state: active` for deprecation is confusing as shown in the example below.

Reply:
My thought is the deprecation section is optional and only added when a feature becomes deprecated. Let me clarify that (and remove the state:active). I removed it from the doc but not from the features yet. I think that section only needs to cover deprecated/retired.

Reply:
Hi Brian, given deprecation is optional, should we remove the deprecation blob in this case as there is no plan to deprecate it?    I think having `state:active` under deprecation is super confusing.